ospf6d/ospf6_top.c

```diff
@@ -124,6 +124,107 @@ struct ospf6 *ospf6_lookup_by_vrf_name(const char *name)
 	return NULL;
 }
 
+/* This is hook function for vrf create called as part of vrf_init */
+static int ospf6_vrf_new(struct vrf *vrf)
+{
+	zlog_debug("%s: VRF Created: %s(%u)", __func__, vrf->name, vrf->vrf_id);
```

Comment:
Protect debug messages with `debug ospf6 <something>`?

Or maybe we don't need these at all.

(same applies to other `zlog_debug` around)